libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.spec.ts

```diff
@@ -45,7 +45,15 @@ describe('Toggle switch component', () => {
       toggleNativeElement = toggleDebugElement.nativeElement;
       toggleInstance = toggleDebugElement.componentInstance;
       testComponent = fixture.debugElement.componentInstance;
-      buttonElement = toggleNativeElement.querySelector('button');
+
+      const button: HTMLButtonElement | null =
+        toggleNativeElement.querySelector('button');
+
+      if (!button) {
```

Comment:
I'm not crazy about this pattern. How horrible would the changes be to make `buttonElement` `HTMLButtonElement | undefined` and then use `buttonElement?.` everywhere it's referenced?

Reply:
Its pretty intensive - I did this in both this and the checkbox because of how many places need to change - but I can do it

Reply:
Done